src/pydrex/minerals.py

```diff
@@ -254,6 +256,18 @@ def __post_init__(self):
             self.orientations_init = Rotation.random(
                 self.n_grains, random_state=self.seed
             ).as_matrix()
+        # For large numbers of grains, the number of ODE's is too large for SciPy.
+        # Instead, we need to solve the banded system of equations.
+        # By default, we use a bandwidth o f 12000 with lband = uband = 6000.
```

Comment:
`ODEs`
`exceeds what LSODA can handle.` instead of `is too large for SciPy.`
`Therefore, we specify the Jacobian matrix as banded.
`of`

Reply:
OK, let me know if 3cdc4b0 is good.